tests/conftest.py

```diff
@@ -236,7 +236,7 @@ def __init__(
         name: str = None,
         imgname: str = None,
         keeptb: bool = False,
-        fakeplatform: str = None,
+        env: list = [],
```

Comment:
what is the reason to change to a list?

Reply:
List type, in case multiple environment variables

Reply:
The parameter _environment_ of **self.client.containers.run** is a **list** type

Reply:
/azpw run